pkg/cloud/network_test.go

```diff
@@ -87,13 +93,21 @@ var _ = Describe("Network", func() {
 		It("does not call to create a new network via GetOrCreateNetwork", func() {
 			ns.EXPECT().GetNetworkID(fakeNetName).Return(fakeNetId, 1, nil)
 			ns.EXPECT().GetNetworkByID(fakeNetId).Return(&cloudstack.Network{Type: isolatedNetworkType}, 1, nil)
+			rs.EXPECT().NewListTagsParams().Return(&cloudstack.ListTagsParams{})
```

Comment:
These expects are not an in-order thing. So, since they're duplicated for each of these tests, we could move the new expects to a `BeforeEach` function.

Reply:
Good plan.  I'll do it.